src/tests/daos_perf.c

```diff
@@ -319,7 +320,8 @@ set_value_buffer(char *buffer, int idx)
 	/* Sets a pattern of Aa, Bb, ..., Yy, Zz, Aa, ... */
 	buffer[0] = 'A' + idx % 26;
 	buffer[1] = 'a' + idx % 26;
-	buffer[TEST_VAL_SIZE - 1] = 0;
+	/* NB: Assume TEST_VAL_SIZE == 4 */
+	*((uint16_t *)&buffer[2]) = 0;
```

Comment:
why not set the 4th char that you added?

Reply:
Because of 4 chars equals one int32_t. For optimization reason I check/copy int32_t instead of looping by chars.

Reply:
i understand, im asking why not setting:
*((uint16_t *)&buffer[3]) = 0;

Reply:
Hmm. Actually the following
*((uint16_t *)&buffer[2]) = 0;
clears both
buffer[2] = 0;
and
buffer[3] = 0;
because of int16_t is 16 bytes size. So, I clear both chars at once.

Reply:
ah yes sorry, my bad. i overlooked the cast there